include/private/tdycharacteristiccurvesimpl.h

```diff
@@ -1,13 +1,21 @@
-#if !defined(TDYSATURATIONIMPL_H)
-#define TDYSATURATIONIMPL_H
+#if !defined(TDYCHARACTERISTICCURVESIMPL_H)
+#define TDYCHARACTERISTICCURVESIMPL_H
 
 #include <petsc.h>
 
+typedef enum {
+  REL_PERM_FUNC_IRMAY=0,
+  REL_PERM_FUNC_MUALEM=1
+} TDyRelPermFuncType;
+
 typedef enum {
   SAT_FUNC_GARDNER=0,
   SAT_FUNC_VAN_GENUCHTEN=1
 } TDySatFuncType;
 
+PETSC_INTERN void RelativePermeability_Mualem(PetscReal,PetscReal,PetscReal*,PetscReal*);
+PETSC_INTERN void RelativePermeability_Irmay(PetscReal,PetscReal,PetscReal*,PetscReal*);
```

Comment:
Change this to `RelPermMualem` and ``RelPermIrmay`

Reply:
On second thought, let's leave it as is and the name will be changed after we have developed a style guide.